README.md

```diff
@@ -1,2 +1,64 @@
-# Cinema app 
-(in progress)
+[![Build Status](https://api.travis-ci.com/den4eg007/taxi-service.svg?branch=master)](https://travis-ci.com/den4eg007/taxi-service)
+## Cinema application
+
+---
+    CinemaApp is an application to manage tickets bought on the occasion of a cinema session by a user. An app can be used
+to organize the interactions between the customer and selling department, a user has an opportunity to choose an
+available sessions, purchasing tickets, make an order, login and  register.
+
+### Usage
+
+---
+<h3> Guest (unauthorized and non registered) can do the following:</h3>
+
+- register
+- login
+- get all available performance sessions according to request
+- view current stages in Database
```

Comment:
viewing all data about movie sessions, movies and cinema halls is available to both USER and ADMIN, but not to users that are not authenticated